lib/src/pdf_page_view.dart

```diff
@@ -70,7 +70,10 @@ class PDFPageView {
     _jsInternal.callMethod('setPdfPage', [pdfPage._jsInternal]);
   }
 
-  void update(num scale, [num rotation = 0]) {
-    _jsInternal.callMethod('update', [scale, rotation]);
+  void update({num scale = 0, num rotation = 0}) {
```

Comment:
Is this a possibility?
```suggestion
  void update(num scale, [num rotation = 0]) {
    // TODO see if we can try/catch if the API is wrong and instead call v2
    _jsInternal.callMethod('update', [scale, rotation]);
  }
  void updateV2({num scale = 0, num rotation = 0}) {
```

Reply:
E.g. in js:
```
document.getDoesNotExist(1, 2)
16:25:07.561 VM202:1 Uncaught TypeError: document.getDoesNotExist is not a function
    at <anonymous>:1:10

try {
  document.getDoesNotExist(1, 2)
} catch (error) {
  document.getElementsByClassName('btn')
}
```

Reply:
I really hope ⏫ is a possibility, would be sweet to not break backwards

Reply:
I couldn't catch an exception unfortunately but I might have something working. I briefly tried it with `v2.5.207` (current wdesk version), `v2.11.338`, and `v2.14.305` (latest). I didn't notice any issues immediately. Let me know what you think